frontend/src/global/constants.ts

```diff
@@ -69,6 +69,7 @@ export const ROUTES = {
     NEW_HOME: '/newconsole',
     NEW_LOGOS: '/newconsole/theming/logos',
     NEW_LOGOS_SIDEBAR: '/newconsole/theming/logos/sidebar',
```

Comment:
@shimulch we can just use the old footer URL to serve the new component, what do you say?

Reply:
@farhaanbukhsh, Could you review my changes? I've used the old footer URL now.